File: product-service/src/main/kotlin/com/lcaohoanq/productservice/repository/OrderDetailRepository.java
package com.lcaohoanq.productservice.repository;

import com.lcaohoanq.productservice.domain.order.OrderDetail;
import java.util.List;
import org.springframework.data.jpa.repository.JpaRepository;

public interface OrderDetailRepository extends JpaRepository<OrderDetail, Long> {

    List<OrderDetail> findByOrderId(Long orderId);

}

File: product-service/src/main/kotlin/com/lcaohoanq/productservice/repository/OrderDetailRepository.kt
package com.lcaohoanq.productservice.repository

import com.lcaohoanq.productservice.domain.order.OrderDetail
import org.springframework.data.jpa.repository.JpaRepository

interface OrderDetailRepository : JpaRepository<OrderDetail, Long> {
    fun findByOrderId(orderId: Long): List<OrderDetail>
}
